x/incentive/keeper/keys.go

```diff
@@ -0,0 +1,124 @@
+package keeper
+
+import (
+	"encoding/binary"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+
+	"github.com/umee-network/umee/v4/util"
+	"github.com/umee-network/umee/v4/x/incentive"
+)
+
+// KVStore key prefixes
+var (
```

Comment:
moved to keeper package (outside of which they should not be used) and lowercased (unexported)